src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -119,7 +119,7 @@ public String getScript() {
             String shell = jenkins.getDescriptorByType(Shell.DescriptorImpl.class).getShellOrDefault(ws.getChannel());
             interpreter = "'" + shell + "' -xe ";
         } else {
-            shf.chmod(0755);
+            interpreter = script.split("[\\r\\n]")[0].replaceAll("#!", '');
```

Comment:
The empty character literal is causing a compilation error.
```suggestion
            interpreter = script.split("[\\r\\n]")[0].replaceAll("#!", "");
```

Reply:
Fixed by 75bdfb1